packages/elements/src/autosuggest/__demo__/index.html

```diff
@@ -652,5 +652,38 @@
         })();
       </script>
     </demo-block>
+    <demo-block layout="normal" header="Accessibility" tags="accessibility">
```

Comment:
Import incorrect renderer in the compound Input demo

Reply:
you can see an error in the console on the demo page.

Reply:
OK

Reply:
done